.gitlab-ci.yml

```diff
@@ -20,11 +20,51 @@ documentation:
       - docu/manual.pdf
       - docu/sphinx/build/html
 
+testing-v7:
+  stage: build
+  tags:
+    - macosx, docker
+  image:
+    name: ${CI_REGISTRY}/internal/docker-igorpro:v7
+  script:
+    - ln -rs procedures "/home/igor/WaveMetrics/Igor Pro 7 User Files/User Procedures/utf"
+    - cp -r docu/examples/Example6 tests
+    - touch tests/DO_AUTORUN.TXT
+    - igorpro execute --verbose --screenshot "tests/Example6.pxp"
+      # generate a proper exit value
+    - exit $(grep "errors\|failures=\"[0-9]\+\"" tests/JU_*.xml | grep -cv "failures=\"0\" errors=\"0\"")
+  artifacts:
+    reports:
+      junit: tests/JU_*.xml
+    paths:
+      - tests/JU_*.xml
+      - screenshot/*.png
+
+testing-v8:
+  stage: build
+  tags:
+    - macosx, docker
+  image:
+    name: ${CI_REGISTRY}/internal/docker-igorpro:v8
```

Comment:
Currently `stable` is the same as `v8` (Igor Pro 8.02) but `stable` will automatically increase to the latest official version available.

Reply:
We could increase v8 to reflect the same status as `stable` in the registry. That would probably be the way to go.

Reply:
Once we have IP9 I want to have an additional job for IP9.